aioftp/client.py

```diff
@@ -562,6 +737,9 @@ class AsyncClientLister(AsyncListerMixin):
                 return cls
 
             async def __anext__(cls):
+                return await cls._anext()
+
+            async def _anext_MSLD_LIST(cls):
```

Comment:
Well, some weird quirk of Python requires that `__anext__` exists before calling `__aiter__`. There may be a solution to this, but I haven't found it